assets/js/phoenix_live_view/live_socket.js

```diff
@@ -400,7 +400,7 @@ export default class LiveSocket {
 
     this.main = this.newRootView(newMainEl, flash, liveReferer)
     this.main.setRedirect(href)
-    this.transitionRemoves()
+    this.transitionRemoves(null, true)
```

Comment:
If you flip the sense of the new parameter to `includingStickies[=false]`, then this invocation can still have empty parameters and the other usage in dom_patch will have two non-default parameter values.